tests/src/services/tiers.service.test.ts

```diff
@@ -304,17 +304,35 @@ describe('TiersService tests', () => {
   });
 
   describe('applyVpnFeatures()', () => {
-    it('When it is called, then it does not throw', async () => {
+    it('When VPN is enabled, then it adds the user tier to the VPN database', async () => {
       const userWithEmail = { ...mocks.mockedUserWithLifetime, email: 'test@internxt.com' };
       const tier = mocks.newTier();
 
       tier.featuresPerService[Service.Vpn].enabled = true;
 
-      const applyVpnFeatures = jest.spyOn(tiersService, 'applyVpnFeatures').mockImplementation(() => Promise.resolve());
+      const addUserTierToVPNDatabase = jest
+        .spyOn(usersService, 'addUserTierToVPNDatabase')
+        .mockImplementation(() => Promise.resolve());
+
+      await tiersService.applyVpnFeatures(userWithEmail, tier);
+
+      expect(addUserTierToVPNDatabase).toHaveBeenCalledWith(
+        userWithEmail.uuid,
+        tier.featuresPerService[Service.Vpn].featureId,
+      );
+    });
+
+    it('When VPN is disabled, then it does not call addUserTierToVPNDatabase', async () => {
```

Comment:
_then it does not send a request to enable a VPN tier_

Again, note that I am not saying _how_ the request is sent. Only talk about: 

**Action > Expected consequence** 

In an understandable, product-oriented language that should be understood by non-technical users also. Thus, if the name of the function or the internal implementation changes, we do not have to change the legend. In fact, the legend should reflect _what's_ going on, not _how_.